src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
Do you want to add `string readGroupSetId` or `array<string> readGroupSedIds`? We need the latter to achieve "multi-BAM splicing". I know it has a cost as @calbach said. I would recommend to keep this part of schema as it is and adopts the google approach in implementation for now.